client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -658,16 +658,42 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         } else if (cellKey === TableHeadersNames.priority) {
             classNames.push(classes.priorityTableCell);
         }
-        if ((isDefaultOrder && !isLoading) &&
-            (rows.length - 1 !== rowIndex) &&
-            rows[rowIndex]?.coronaTestDate &&
-            (getFormattedDate(rows[rowIndex]?.coronaTestDate) !== getFormattedDate(rows[rowIndex + 1]?.coronaTestDate))) {
+
+        if(isLastNestedCell === undefined) {
```

Comment:
isLastNestedCell is boolean type so why not passing false instead

Reply:
isLastNestedCell is either boolean if this is a nested cell or undefined if the cell is not a nested cell , when I think about it it's not really readable so I moved the condition to a constant [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/962/commits/02f53efcb970029297aafe050a33993d62075c70)